Doc/library/stdtypes.rst

```diff
@@ -2760,12 +2762,14 @@ arbitrary binary data.
             bytearray.decode(encoding="utf-8", errors="strict")
 
    Return a string decoded from the given bytes.  Default encoding is
```

Comment:
```suggestion
   Return the bytes decoded to a :class:`str`.
```

As above:
* Following the original issue, keep the summary line in its own paragraph for clarity and readability.
* Per the original issue, properly markup/link the `str` object.
* Also, per the general principles of reference type documentation, keep this more concise.